src/sentry/replays/lib/event_linking.py

```diff
@@ -17,7 +17,7 @@ class EventLinkKafkaMessage(TypedDict):
     replay_id: str
     project_id: int
     segment_id: None
-    payload: list[int]
+    payload: str
```

Comment:
I almost think this should be a union between the various payload dictionaries that are expected on the rust size -- smuggling through bytes as `list[int]` and double-json encoding both seem like hacks